docs/graphback-datasync.md

```diff
@@ -0,0 +1,235 @@
+---
+id: graphback-datasync
+title: Data Synchronization
+sidebar_label: Setting up Data Synchronization using Graphback
```

Comment:
```suggestion
sidebar_label: Setting up Data Synchronization
```

Current form is a bit long:

![image](https://user-images.githubusercontent.com/11743717/85002965-7878c400-b14d-11ea-92ef-63fd9a11e5be.png)